docs/en/connector-v2/sink/Kafka.md

```diff
@@ -120,6 +120,63 @@ sink {
 }
 ```
 
+### AWS MSK SASL/SCRAM
+
+Replace the following `${username}` and `${password}` with the configuration values in AWS MSK.
+
+```hocon
+sink {
+  kafka {
+      topic = "seatunnel"
+      bootstrap.servers = "localhost:9092"
+      partition = 3
+      format = json
+      kafka.request.timeout.ms = 60000
+      semantics = EXACTLY_ONCE
+      kafka.security.protocol=SASL_SSL
+      kafka.sasl.mechanism=SCRAM-SHA-512
+      kafka.sasl.jaas.config="org.apache.kafka.common.security.scram.ScramLoginModule required \nusername=${username}\npassword=${password};"
+  }
+  
+}
+```
+
+### AWS MSK IAM
+
+Download `aws-msk-iam-auth-1.1.5.jar` from https://github.com/aws/aws-msk-iam-auth/releases and put it in `$SEATUNNEL_HOME/plugin/kafka/lib` dir.
+
+Please ensure the IAM policy have `"kafka-cluster:Connect",`. Like this:
+
+
+```hocon
+"Effect": "Allow",
+"Action": [
+    "kafka-cluster:Connect",
+    "kafka-cluster:AlterCluster",
+    "kafka-cluster:DescribeCluster"
+],
+```
+
```

Comment:
TBH, I don't know why we are adding these things, it has nothing to do with the project.